Bandwidth.Iris/Model/Order.cs

```diff
@@ -26,9 +26,9 @@ public static Task<OrderResult> Get(Client client, string id)
             return client.MakeGetRequest<OrderResult>(client.ConcatAccountPath(OrderPath), null, id);
         }
 
-        public static async Task<OrderResult[]> List(Client client, IDictionary<string, object> query = null)
+        public static async Task<OrderListResponse> List(Client client, IDictionary<string, object> query = null)
```

Comment:
Can you explain to me how this does not result in a breaking change? I'm not the most experienced with strongly typed languages but this looks like it could be a breaking change